presto-spark-base/src/main/java/com/facebook/presto/spark/execution/PrestoSparkRowBatch.java

```diff
@@ -43,25 +43,27 @@
     private final int rowCount;
     private final byte[] rowData;
     private final int[] rowPartitions;
-    private final int[] rowSizes;
+    private final int[] rowOffsets;
+    private final int totalSize;
     private final long retainedSizeInBytes;
 
-    private PrestoSparkRowBatch(int partitionCount, int rowCount, byte[] rowData, int[] rowPartitions, int[] rowSizes)
+    private PrestoSparkRowBatch(int partitionCount, int rowCount, byte[] rowData, int[] rowPartitions, int[] rowOffsets, int totalSize)
```

Comment:
entryData, entryOffsets, entryPartitions

Reply:
I deliberately didn't want to do the extensive renames, as this might potentially go to a very large scope. Since this code is transitional I would like to keep it the way it is, as then when we revert this patch we would have to rename everything back.